js/test/strings-ext/testlistfmt_mt_MT.js

```diff
@@ -0,0 +1,253 @@
+        test.done();
+    },
+    testListFmtUnitStyleNumberFormatOneFull_mt_MT: function(test) {
+        var fmt = new ListFmt({
+            locale: "mt-MT",
+            style: "unit",
+            length: "full"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["waħda"]), "waħda");
+        test.done();
+    },
+    testListFmtUnitStyleNumberFormatTwoFull_mt_MT: function(test) {
+        var fmt = new ListFmt({
+            locale: "mt-MT",
+            style: "unit",
+            length: "full"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["waħda", "tnejn"]), "waħda u tnejn");
+        test.done();
+    },
+    testListFmtUnitStyleNumberFormatThreeFull_mt_MT: function(test) {
+        var fmt = new ListFmt({
+            locale: "mt-MT",
+            style: "unit",
+            length: "full"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["waħda", "tnejn", "tlieta"]), "waħda, tnejn, u tlieta");
+        test.done();
+    },
+    testListFmtUnitStyleNumberFormatFourFull_mt_MT: function(test) {
+        var fmt = new ListFmt({
+            locale: "mt-MT",
+            style: "unit",
+            length: "full"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["waħda", "tnejn", "tlieta", "erbgħa"]), "waħda, tnejn, tlieta, u erbgħa");
+        test.done();
+    },
+    testListFmtUnitStyleNumberFormatFiveFull_mt_MT: function(test) {
+        var fmt = new ListFmt({
+            locale: "mt-MT",
+            style: "unit",
+            length: "full"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["waħda", "tnejn", "tlieta", "erbgħa", "ħamsa"]), "waħda, tnejn, tlieta, erbgħa, u ħamsa");
+        test.done();
+    },
+    testListFmtORStyleNumberFormatOne_mt_MT: function(test) {
+        var fmt = new ListFmt({
+            locale: "mt-MT",
+            style: "disjunction"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["waħda"]), "waħda");
+        test.done();
+    },
+    testListFmtORStyleNumberFormatTwo_mt_MT: function(test) {
+        var fmt = new ListFmt({
+            locale: "mt-MT",
+            style: "disjunction"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["waħda", "tnejn"]), "waħda or tnejn");
+        test.done();
+    },
+    testListFmtORStyleNumberFormatThree_mt_MT: function(test) {
+        var fmt = new ListFmt({
+            locale: "mt-MT",
+            style: "disjunction"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["waħda", "tnejn", "tlieta"]), "waħda, tnejn, or tlieta");
```

Comment:
according to google translate, the translation of "or" in Maltese is "jew"

Reply:
I've manually updated a list.json file to support it. CLDR says `or`